### PR TITLE
task: Yarn v4

### PR DESCRIPTION
**Pull request by chriswk**
Trying again, this time with correct .gitignore already setup, and a workflow configured to try what was failing prior to our revert.

### CODE REVIEW

File: docker/.yarn/releases/yarn-4.2.2.cjs
Comment (alvinometric) [inline docker/.yarn/releases/yarn-4.2.2.cjs:1]: Should this file be there or should it be ignored?

File: frontend/.yarn/releases/yarn-4.3.0.cjs
Comment (alvinometric) [inline frontend/.yarn/releases/yarn-4.3.0.cjs:1]: Same for this file?

File: website/.yarn/releases/yarn-4.3.0.cjs
Comment (alvinometric) [inline website/.yarn/releases/yarn-4.3.0.cjs:1]: Same for this file

Comment (chriswk) [reply website/.yarn/releases/yarn-4.3.0.cjs:1]: Yes, based on our .gitignore, the files inside .yarn/releases should be included

**Review by gastonfournier — APPROVED** [review]
:+1:

**Pull request merged**